www/js/lib/uiUtil.js

```diff
@@ -32,6 +32,38 @@ if (webpMachine) {
 }
 
 define(rqDef, function(settingsStore) {
+
+    /**
+     * Displays a Bootstrap alert or confirm dialog box depending on the options provided
+     * @param {*} node 
+     * @param {*} nodeAttribute 
+     * @param {*} content 
+     * @param {*} mimeType 
+     * @param {*} callback 
+     */
+    function systemAlert(label, message, isConfirm){
+        return new Promise(function(resolve, reject) {
```

Comment:
Space after `function` (because it's anonymous).
Codefactor is warning that `reject` is defined but not used. Is there a possibility of this function producing an error and therefore rejecting? Maybe you should have a reject block just to appease codefactor!

Reply:
Calling the reject method when the `systemAlert` function is called without the `label` or `message` parameters.